parsons/google/google_bigquery.py

```diff
@@ -1018,6 +1018,26 @@ def get_columns_list(self, schema: str, table_name: str) -> list:
 
         return [x for x in first_row.columns]
 
+    def get_row_count(self, schema: str, table_name: str) -> int:
```

Comment:
Kind of a bummer that you have to fetch the whole table in order to get the number of rows! But presumably there's not a better way to do this.

Reply:
I believe this is more memory efficient than `SELECT COUNT(*) FROM {{ table }}` but if others have thoughts here we should update this accordingly

Reply:
I'd think that doing a `SELECT COUNT(*)` would be more efficient? If we select the whole table, then it needs to transfer all of the data from the database to the computation site over the network. If we do a `SELECT COUNT(*)` then it just needs to transfer the single integer.

Reply:
You're right, I assumed that running `select count(*)` would scan the table in its entirety but it doesn't! I'll change this

Reply:
Pushed! Still need to add a formal unit test before merging but this approach does seem more efficient

<img width="230" alt="image" src="https://github.com/move-coop/parsons/assets/47256454/2c269dae-8622-4cc5-80e9-e6a8ddb265ef">